src/main/java/com/autilus/hw0306/task2/App.java

```diff
@@ -0,0 +1,19 @@
+package com.autilus.hw0306.task2;
+
+import java.util.logging.*;
```

Comment:
doesn't use auto import

Reply:
I change, but why I can't use it?)

Reply:
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad